repos/system_upgrade/common/actors/systemd/checksystemdbrokensymlinks/libraries/checksystemdbrokensymlinks.py

```diff
@@ -0,0 +1,106 @@
+import os
+
+from leapp import reporting
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.stdlib import api
+from leapp.models import SystemdBrokenSymlinksSource, SystemdServicesInfoSource
+
+FMT_LIST_SEPARATOR = '\n    - '
+
+
+def _report_broken_symlinks(symlinks):
+    summary = (
+        'Leapp detected broken systemd symlinks on the system that do not'
+        ' correspond to any installed systemd unit.'
+        ' This typically happens when the original systemd unit file has been'
+        ' removed (e.g. an rpm removal) or renamed and the system configration'
+        ' has not been properly modified.'
+        ' These symlinks will not be handled during the in-place upgrade'
+        ' as they are already broken.'
+        ' The list of detected broken systemd symlinks:{}{}'
+        .format(FMT_LIST_SEPARATOR, FMT_LIST_SEPARATOR.join(sorted(symlinks)))
+    )
+
+    command = ['rm'] + symlinks
```

Comment:
Is not "rm" command asking for confirmation under certain circumstances? In case of the Cockpit/Satellite solution this would end up being stuck waiting for input. What about adding "-f" option?

Reply:
let's put there '/usr/bin/rm' which resolve the possible issue.